types.d.ts

```diff
@@ -0,0 +1,3 @@
+import { Application } from "stimulus";
```

Comment:
Could you use single quotes? That's the coding styles used on Symfony and UX.

That makes me think we should enable prettier on this repo :) .